tasks/main.yml

```diff
@@ -10,6 +10,3 @@
 - include: config.yml
```

Comment:
I would use `import_tasks`, as in here: https://github.com/idealista/airflow-role/blob/develop/tasks/main.yml

Reply:
done